src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -89,10 +93,19 @@
         handler=StringArrayOptionHandler.class)
     List<String> externs = new ArrayList<>();
 
+    @Option(name = "--depgraphs",
+        usage = "file(s) which contains the JSON representation of the dependency graph",
+        metaVar = "file.depgraph...",
+        handler=StringArrayOptionHandler.class)
```

Comment:
yep but no indentation around the equals sign.

Don't you have an automated formatter?